KrystianKjjk/CodersCamp2020.Project.FullStack-Node-React.OrganizationApp
# Fs password reset

to test
- make sure your frontend runs on localhost:3000 (url is set on the backend rigidly and would have to be changed to the real one later)
- run npm start
- go to forgot your password
- provide your email
- click on send
- wait till the email is delivered (might take a while, not sure why - probably email provider lagging) - send the email just once as every next attempt resets the token, so you have to use the last email anyway :)
- click the link (edit the url to http afterwards as localhost is running on http - final url will be https and the protocol is rigid on the backend)
- provide new password
- click send
- use your new password :)
